lib/Message.php

```diff
@@ -318,4 +320,23 @@ function getHttpVersion() {
         return $this->httpVersion;
 
     }
+
+    /**
+     * Runs given callback and captures data sent to php://output stream.
+     *
+     * @param callable $callback
+     * @throws \RuntimeException when ob_start() fails to start output buffer
+     * @return string
+     */
+    private function captureCallbackOutput($callback)
+    {
+        $success = ob_start();
+        if ($success === false) {
```

Comment:
You don't really need this check. I've never encountered a situation where this can possibly fail.